.github/workflows/test-node.yaml

```diff
@@ -1,9 +1,12 @@
 name: Frontend Test
 
 on:
-  - push
   - pull_request
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
```

Comment:
@andreyvelich I'm not fully in sync with why we need to use `concurrency`. Could you provide some more context?

Reply:
It is useful when the developer push multiple commits on an open PR in short period.
https://github.com/kubeflow/katib/issues/2024#issuecomment-1320524310

To cancel previous-running workflows and run with new source codes.

Reply:
Yes, @anencore94 is right, when the newer commit has been pushed, previous workflows are canceled.
@kimwnasptd You can check example here: https://github.com/kubeflow/katib/actions/runs/3534409566